Makefile

```diff
@@ -18,7 +18,11 @@ TIMEOUT ?= 14400s
 # TEST_EXEC_NODES=1, otherwise by default the specs are run in parallel on 4 ginkgo test node.
 # NOTE: Any TEST_EXEC_NODES value greater than one runs the spec in parallel
 # on the same number of ginkgo test nodes.
-TEST_EXEC_NODES ?= 4
+if $(CLUSTER_ON_IBM_CLOUD)
+   TEST_EXEC_NODES ?= 24
+else
+   TEST_EXEC_NODES ?= 2
```

Comment:
Why reduce from 4 to 2? https://github.com/openshift/odo/pull/4811 moved it from 2 to 4. Why do you suggest we go back to using 2?

Reply:
my mistake, changing it back to 4